### ISSUE TITLE
[APP] Ollama: logs and models stay untouched

### ISSUE THREAD

**Issue by hermannsorgel** [alienator88/Pearcleaner#66]
**Describe the app**

Ollama, a locall LLMs runner, keeps logs and models in the ~/.ollama. Pearcleaner does not suggest removing this folder when uninstalling Ollama.

**App Details**
1. Name: Ollama
2. Version: 0.1.41
3. Source: [website](https://ollama.com/download)
4. Type: Regular
5. PATH/S to the files/folders not being deleted
~/.ollama

**Desktop :**
 - OS: 14.5
 - Pearcleaner App Mode: Regular
 - Pearcleaner Version: 3.7.2

**Screenshots**
<img width="400" alt="'Pearcleaner screenshot'" src="https://github.com/alienator88/Pearcleaner/assets/25165356/f40d7a5c-d7b4-4192-afad-ab0739c8081a">

``` bash
du -hd 1 .ollama
4.3G	.ollama/models
 88K	.ollama/logs
4.3G	.ollama
```

**Additional context**

Thank you for a nice app.

**Comment by alienator88** [comment]
Hi, I looked at the code and it looks like the home directory is not part of the search Locations the app goes through.
I've gone ahead and added that in for the next release, which should find the folder you mentioned once you update to that version.
Thanks!

**Comment by alienator88** [comment]
Released in v3.7.3
